code/game/objects/effects/decals/Cleanable/misc.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "Someone should clean that up."
 	gender = PLURAL
 	density = 0
-	layer = 2
+	layer = ABOVE_OPEN_TURF_LAYER

```

Comment:
These files make me want to drink. byond is OB JECT ORI ENTED PEOPLE YOU DO N"T NEED TO COPYPASTE ALL THESE